NEWS.md

```diff
@@ -24,6 +24,8 @@ New library functions
 Standard library changes
 ------------------------
 
+* `complex(::Array, ::Array)` method has been added to maintain consistency with
+  `complex(::Array)` ([#31676]).
```

Comment:
This should normally reference the PR, i.e. 31710

Reply:
I would also say `complex(array, array)` and `complex(array)` here, since it's not actually restricted to the `Array` type, and just saying `array` will be clear enough for NEWS.